src/main/java/net/william278/velocitab/config/Settings.java

```diff
@@ -16,37 +22,84 @@
+    private Map<String, String> footers = Map.of("default", "[There are currently %players_online%/%max_players_online% players online](gray)");
+    @YamlKey("formats")
+    @YamlComment("How the player will appear on the tab list per group.")
+    private Map<String, String> formats = Map.of("default", "&7[%server%] &f%prefix%%username%");
+    @YamlKey("server_groups")
+    @YamlComment("The servers in each group\nAll servers not defined in a group will be excluded from Velocitab")
+    private Map<String, List<String>> serverGroups = Map.of("default", List.of("lobby1", "lobby2", "lobby3"));
+    @YamlKey("fallback_group")
+    @YamlComment("The group of servers that not defined in server_groups. \"excluded\" will exclude them from Velocitab.")
+    private String fallbackGroup = "default";
+    @YamlKey("update_rate")
     private int updateRate = 0;
 
-    private Settings() {
+    public Settings(@NotNull Velocitab plugin) {
+        this.serverGroups = Map.of("default",
+                plugin.getServer().getAllServers().stream().map(server -> server.getServerInfo().getName()).toList()
+        );
     }
 
     @NotNull
-    public String getHeader() {
-        return StringEscapeUtils.unescapeJava(header);
+    public String getHeader(String serverGroup) {
+        return StringEscapeUtils.unescapeJava(
+                headers.getOrDefault(serverGroup, "&rainbow&Running Velocitab v" + BuildConstants.VERSION + " by William278"));
     }
 
     @NotNull
-    public String getFooter() {
-        return StringEscapeUtils.unescapeJava(footer);
+    public String getFooter(String serverGroup) {
+        return StringEscapeUtils.unescapeJava(
+                footers.getOrDefault(serverGroup, "[There are currently %players_online%/%max_players_online% players online](gray)"));
     }
 
     @NotNull
-    public String getFormat() {
-        return StringEscapeUtils.unescapeJava(format);
+    public String getFormat(String serverGroup) {
+        return StringEscapeUtils.unescapeJava(
+                formats.getOrDefault(serverGroup, "&7[%server%] &f%prefix%%username%"));
     }
 
-    public boolean isServerExcluded(@NotNull String serverName) {
-        return excludedServers.contains(serverName);
+    /**
+     * Get the server group that a server is in
+     *
+     * @param serverName The name of the server
+     * @return The server group that the server is in, or "default" if the server is not in a group
+     */
+    public String getServerGroup(String serverName) {
+        return serverGroups.entrySet().stream()
+                .filter(entry -> entry.getValue().contains(serverName)).findFirst()
+                .map(Map.Entry::getKey).orElse(fallbackGroup);
+    }
+
+    @NotNull
+    public Optional<List<String>> getBrotherServers(String serverName) {
+        return serverGroups.entrySet().stream()
+                .filter(entry -> !entry.getKey().equals(fallbackGroup))// Exclude the fallback group
```

Comment:
I'd like to see these comments moved to a JavaDoc comment above the method rather than in the logic